app/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :age, :name, :parent_permission
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?(age)
+    is_of_age(age) && @parent_permission
```

Comment:
- [ ] The method that you defined was `of_age` but you call `is_of_age` and is not defined, kindly fix that 🙃 
- [ ] Also the public method `can_use_services?` should return true if person is of age or if they have permission from parents so you should use the `or/||` symbol and not the `and/&&` symbol. The code can be similar to: `of_age?() || @parent_permission`